src/lib/draw-and-pick.js

```diff
@@ -23,6 +23,12 @@ import {GL, glContextWithState} from 'luma.gl';
 import {getUniformsFromViewport} from './viewport-uniforms';
 import {log, getBlendMode, setBlendMode} from './utils';
 
+const MOTION_EVENTS = [
+  'dragmove',
```

Comment:
Sorry, I don't understand -- I'm not making a distinction between them, they're both included here...?